builtin/providers/cloudstack/resource_cloudstack_loadbalancer_rule.go

```diff
@@ -215,6 +228,41 @@ func resourceCloudStackLoadBalancerRuleUpdate(d *schema.ResourceData, meta inter
 				"Error updating load balancer rule %s", name)
 		}
 	}
+
+	if d.HasChange("member_ids") {
+		o, n := d.GetChange("member_ids")
+		ombs, nmbs := o.(*schema.Set), n.(*schema.Set)
+
+		setToStringList := func(s *schema.Set) []string {
+			l := make([]string, s.Len())
+			for i, v := range s.List() {
+				l[i] = v.(string)
+			}
+			return l
+		}
+
+		membersToAdd := setToStringList(nmbs.Difference(ombs))
+		membersToRemove := setToStringList(ombs.Difference(nmbs))
+
+		log.Printf("[DEBUG] Members to add: %v, remove: %v", membersToAdd, membersToRemove)
+
+		if len(membersToAdd) > 0 {
+			ap := cs.LoadBalancer.NewAssignToLoadBalancerRuleParams(d.Id())
```

Comment:
_nitting_ can you make that just `p` instead of `ap`? The `ap` used in the create function is only used because `p` was already used in the same scope...